src/Tribe/Service_Providers/Context.php

```diff
@@ -259,6 +259,15 @@ static function ( $data ) {
 				],
 				'tec_post_type'        => [
 					'read' => [
+						Tribe__Context::FUNC          => [
+							static function () {
+								return count( array_filter( [
+									! empty( tribe_get_request_var( TEC::POSTTYPE, false ) ),
+									! empty( tribe_get_request_var( Venue::POSTTYPE, false ) ),
+									! empty( tribe_get_request_var( Organizer::POSTTYPE, false ) ),
+								] ) );
```

Comment:
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: a18c426a3c6acbe893873f97132c28b6</sub>